cumulusci/tasks/connectedapp.py

```diff
@@ -0,0 +1,176 @@
+import json
+import re
+import os
+from cumulusci.core.config import ServiceConfig
+from cumulusci.core.exceptions import TaskOptionsError, ServiceNotConfigured
+from cumulusci.core.utils import process_bool_arg
+from cumulusci.tasks.sfdx import SFDXBaseTask, SFDX_CLI
+from cumulusci.utils import random_alphanumeric_underscore
+from cumulusci.utils import temporary_dir
+
+CONNECTED_APP = """<?xml version="1.0" encoding="UTF-8"?>
+<ConnectedApp xmlns="http://soap.sforce.com/2006/04/metadata">
+    <contactEmail>{email}</contactEmail>
+    <label>{label}</label>
+    <oauthConfig>
+        <callbackUrl>http://localhost:8080/callback</callbackUrl>
+        <consumerKey>{client_id}</consumerKey>
+        <consumerSecret>{client_secret}</consumerSecret>
+        <scopes>Web</scopes>
+        <scopes>Full</scopes>
+        <scopes>RefreshToken</scopes>
+    </oauthConfig>
+</ConnectedApp>"""
+
+PACKAGE_XML = """<?xml version="1.0" encoding="UTF-8"?>
+<Package xmlns="http://soap.sforce.com/2006/04/metadata">
+    <types>
+        <members>*</members>
+        <name>ConnectedApp</name>
+    </types>
+    <version>44.0</version>
+</Package>"""
+
+
+class CreateConnectedApp(SFDXBaseTask):
+    client_id_length = 85
+    client_secret_length = 32
+    deploy_wait = 5
+    task_options = {
+        "label": {
+            "description": "The label for the connected app.  Must contain only alphanumeric and underscores",
+            "required": True,
+        },
+        "email": {
+            "description": "The email address to associate with the connected app.  Defaults to email address from the github service if configured.",
+            "required": True,
```

Comment:
This means if you don't specify an email it'll break before it gets to the point where it tries to use the email from the github service, no?